database/entity/0072-add_communityuuid_to_transactions_table/Transaction.ts

```diff
@@ -0,0 +1,163 @@
+/* eslint-disable no-use-before-define */
+import { Decimal } from 'decimal.js-light'
+import { BaseEntity, Entity, PrimaryGeneratedColumn, Column, OneToOne, JoinColumn } from 'typeorm'
+import { DecimalTransformer } from '../../src/typeorm/DecimalTransformer'
+import { Contribution } from '../Contribution'
+import { DltTransaction } from '../DltTransaction'
+
+@Entity('transactions')
+export class Transaction extends BaseEntity {
+  @PrimaryGeneratedColumn('increment', { unsigned: true })
+  id: number
+
+  @Column({ type: 'int', unsigned: true, unique: true, nullable: true, default: null })
```

Comment:
Attention: 
nullable = true + unique = true:  means that only one entry with value NULL allowed to exist in db

Reply:
seemingly not, beause this column definition still exists for a longer time.
In production database the table `transactions` contains for the column `previous` several entries with value NULL.
The definition `unique: true` exists only on entity-level, in database there should exist an unique-key-contraint and the constraints have to be actived to reach your described effect.

I don't know if on entity-level the `unique:true` will have any effect, without dedicated database support...